chapters/Index.md

```diff
@@ -35,24 +35,2394 @@ Jintek           <info@jintek.consulting>
+
+The user-defined index allows any user to stand up a custom test network and create **did:btcr2** identifiers on it. However, anyone encountering such an identifier would have to know the details of the network (e.g., challenge and seed node for signet) to use it. This means that:
+
+* The interpretation of a user-defined index is by mutual agreement between parties issuing **did:btcr2** identifiers and those resolving them.  
+* Other users may use the same index, and **did:btcr2** identifiers will **not** be interoperable in each other's network.
+
+When the last part of the encoding is of a `key-value`, the Human Readable Part (HRP) of the Bech32 encoding is set to `k`. When the last part of the encoding is of a `hash-value`, the HRP is set to `x`. The HRP is followed by a separator which is always `1`, which is then followed by the `bech32-encoding`.
+
+The Augmented Backus-Naur (ABNF) for a **did:btcr2** identifier is as follows:
+
+```
+did-btcr2 = "did:btcr2:" id-bech32
+id-bech32 = key-encoding / hash-encoding
+hash-encoding = "x1" bech32-encoding
+key-encoding = "k1" bech32-encoding
+bech32-encoding = *bech32char
+bech32char = "0" / "2" / "3" / "4" / "5" / "6" / "7" / "8" / "9" / "a" / "c" / 
+"d" / "e" / "f" / "g" / "h" / "j" / "k" / "l" / "m" / "n" / "p" / "q" / "r" / 
+"s" / "t" / "u" / "v" / "w" / "x" / "y" / "z" 
+```
+
+ABNF is defined by [Internet Engineering Task Force (IETF) RFC5234](https://datatracker.ietf.org/doc/html/rfc5234).
+
+For details on how to interpret a BTCR2 DID, see \[Resolve\].
+
+## Update Distribution
+
+When additional data is required for resolution, the controller has two primary means for getting that data to the Resolver, both secured by cryptographic hash. The first is using ::Sidecar::. The second is ::Content Addressable Storage:: (CAS). No matter, how the Resolver gets this data, all update data must be available to process resolution. We use cryptographic hashes to ensure that the data received by the Resolver is the data secured by a legitimate update.
+
+All files used for resolution are identified by their SHA256 cryptographic hash calculated according to the \[JSON Canonicalization and Hash\] algorithm.
+
+As a resolver goes through the resolution process, it encounters one or more document hashes, which it uses to identify the files of interest.
+
+While it's possible for a single **did:btcr2** identifier to mix the two distribution mechanisms, it is not recommended. Instead, it is RECOMMENDED that controllers pick a distribution mechanism and use it throughout the lifetime of a given DID.
+
+### Sidecar
+
+::Sidecar:: provides  alongside the **did:btcr2** identifier being resolved. This is analogous to a sidecar on a motorcycle bringing along a second passenger: the DID controller provides the DID document history (in the form of JSON Patch transformations) alongside the DID to the relying party so that the resolver can construct the DID document.
```

Comment:
```suggestion
::Sidecar:: provides  alongside the **did:btcr2** identifier being resolved. This is analogous to a sidecar on a motorcycle bringing along a second passenger: the DID controller provides the DID document history (in the form of ::BTCR2 Updates:: and any additional proofs) alongside the DID to the relying party so that the resolver can construct the DID document.
```

Not sure about this one. Sidecar Data can include:

- BTCR2 Updates
- Beacon Announcement Maps
- SMT Proofs
- Genesis Document

But how specific should we be in this section